cluster-sync/sync.sh

```diff
@@ -123,6 +123,16 @@ function wait_cdi_pods_updated {
   fi
 }
 
+function dump_upgrade_info {
+  echo "Dumping upgrade object information"
+  _kubectl get all --namespace cdi-testing-old-version-artifacts
+  _kubectl describe pvc olddv-v1alpha1 --namespace cdi-testing-old-version-artifacts
+  _kubectl describe pvc olddv-v1beta1 --namespace cdi-testing-old-version-artifacts
+  _kubectl get dv --namespace cdi-testing-old-version-artifacts -o yaml
```

Comment:
Just a nitpick, wouldn't `_kubectl get dv -o yaml` and `_kubectl describe dv` output too much redundant information? Shouldn't `describe` be enough?

Reply:
So a describe will output the relevant events, and part of the yaml, the -o yaml outputs the entire yaml. There is definitely some overlapping information, but when debugging its usually better to have too much information instead of not enough. I can take the -o yaml out, its not a huge deal.

Reply:
No problem, I guess in case of error it's just better to have all information possible.